libs/ui/src/lib/table-responsive/table-responsive.component.ts

```diff
@@ -0,0 +1,34 @@
+import {
+  ChangeDetectionStrategy,
+  Component,
+  HostBinding,
+  Input,
+} from '@angular/core';
+import { Gap } from '../track/track';
+
+const statuses = [
+  {
+    severity: 'info',
+    label: 'Lõppenud',
+  },
+  {
+    severity: 'error',
+    label: 'Katkenud',
+  },
+];
+@Component({
+  selector: 'cvi-ng-table-responsive',
+  templateUrl: './table-responsive.component.html',
+  changeDetection: ChangeDetectionStrategy.OnPush,
+})
+export class TableResponsiveComponent {
+  @Input() data!: any[];
+  @Input() headerLabels!: any[];
+  @Input() cardRowGap: Gap = 1;
```

Comment:
Should this be a prop? It is a design detail that is unlikely to change.

Reply:
Refactored, I agree. Overlooked other new components (card and labeled items) too.